[Fix]Fix LocalFile doc (#7887) and Fix ClickHouse Connector (#7897)

YOMO-Lee opened the pull request:
1. Added file filtering instructions to the **localfile** connector documentation
2. Fixed the bug where **clickhouse** connectors cannot stop properly under multiple parallelism

YOMO-Lee committed:
[Fix][Doc] Fix LocalFile doc (#7887)

Supplement and optimize the description of the LocalFile connector on filtering files
[(#7887)](https://github.com/apache/seatunnel/issues/7887)

YOMO-Lee committed:
Merge branch 'apache:dev' into dev

YOMO-Lee committed:
Merge branch 'apache:dev' into dev

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner,  Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner,  Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner, Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```

YOMO-Lee committed:
[Fix][Doc] Fix LocalFile doc (#7887)

Continue to optimize the document about filtering files and add some examples
[(#7887)](https://github.com/apache/seatunnel/issues/7887)

YOMO-Lee committed:
Merge branch 'apache:dev' into dev

Hisoka-X commented on seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java:
```diff
@@ -101,6 +101,13 @@ public void testClickhouse(TestContainer container) throws Exception {
 
+    @TestTemplate
+    public void testSourceParallelism(TestContainer container) throws Exception {
+        LOG.info("=========多并行度测试===========");
```
please remove chinese log.

YOMO-Lee replied:
Okay, it's done

YOMO-Lee committed:
[Fix][Doc] Fix LocalFile doc (#7887)

Change to English log
[(#7887)](https://github.com/apache/seatunnel/issues/7887)

YOMO-Lee committed:
Merge remote-tracking branch 'origin/dev' into dev
